### PR TITLE
Use EntityIdentifier on Environment Field for Application

### PR DESCRIPTION
**Pull request by anna-cross**
Changing type of Environment field for Application to use EntityIdentifier

### CODE REVIEW

File: pkg/meroxa/application.go [hunk 1]
```diff
@@ -50,20 +44,20 @@ type EntityDetails struct {
...
-	Deployments []EntityIdentifier     `json:"deployments,omitempty"`
-	CreatedAt   time.Time              `json:"created_at"`
-	UpdatedAt   time.Time              `json:"updated_at"`
-	DeletedAt   time.Time              `json:"deleted_at,omitempty"`
+	UUID        string                `json:"uuid"`
+	Name        string                `json:"name"`
+	Language    string                `json:"language"`
+	GitSha      string                `json:"git_sha,omitempty"`
+	Status      ApplicationStatus     `json:"status,omitempty"`
+	Environment EntityIdentifier      `json:"environment,omitempty"`
```
Comment (dianadoherty) [inline pkg/meroxa/application.go:52]:
```suggestion
	Environment *EntityIdentifier      `json:"environment,omitempty"`
```

Comment (anna-cross) [reply pkg/meroxa/application.go]: Added!

File: Makefile [hunk 2]
```diff
@@ -16,3 +16,17 @@ gomod:
 .PHONY: test
 test: mockgen
 	go test ./...
+
+.PHONY: lint
+lint: ./bin/golangci-lint
```
Comment (dianadoherty) [inline Makefile:21]: Can you add the /bin folder to gitignore? And then removing the file `bin/golangci-lint` from the PR?

Comment (anna-cross) [reply Makefile:21]: Yup! Removed!

**Review by dianadoherty — APPROVED** [review]

**Pull request merged**